### ISSUE TITLE
Twitter account

### ISSUE THREAD

**Issue by pwnorbitals** [cuberite/cuberite#4954]
I see the twitter account (https://twitter.com/cuberiteproject) has been inactive for quite some time (last post from 2017).
I suggest the account should be closed if deemed useless, or resume maintaining it if it really has value (which isn’t the case imo...)

**Comment by tigerw** [comment]
:O

first Discord, now Twitter, there are a lot of channels I'm unaware of...

**Comment by bearbin** [comment]
Youtube as well! The twitter account is a bit pointless, but I'll try and keep it updated whenever we have a new news entry.

**Comment by pwnorbitals** [comment]
Well I advocate that having no account is better than an unmaintained one (which could degrade the image of the project)
If the intent is to update it, then it’s awesome !

**Comment by wereii** [comment]
Have a new release ci pipeline push a tweet about it 🙃

**Comment by 12xx12** [comment]
we don't do releases here

**Comment by koutsie** [comment]
> Have a new release ci pipeline push a tweet about it

New releases defo should be tweeted out.

**Comment by tigerw** [comment]
Yep, I think we need to work on our community outreach. Especially as we're replicating an existing game's _server_, building excitement and attracting contributors is always going to be a challenge. For the project to have long term success I think getting the word out is as important as the technical implementation

**Comment by 12xx12** [comment]
Does anyone know of a bot that can post if there was a new CI build?

**Comment by wereii** [comment]
What about Github Actions? 
https://github.com/marketplace/actions/twitter-action

**Comment by koutsie** [comment]
> Does anyone know of a bot that can post if there was a new CI build?
> What about Github Actions?
> https://github.com/marketplace/actions/twitter-action

That's how you do it :)
https://github.com/marketplace/actions/send-tweet-action for example

I veto doing this :+1:

**Comment by wereii** [comment]
Back when I jokingly noted about tweets (and @12xx12 also had a note there before editing it :upside_down_face: ) the problem was/is that cuberite does not really have a notion of releases as it's tracking features of vanilla, not making new features on it's own.

Unless you want to spam every new commit in master to twitter then one idea would be to have everything "hot" go into `devel` branch and only do devel -> master once in a while (after ""testing"" it) and push that as a tweet.

**Comment by koutsie** [comment]
> Back when I jokingly noted about tweets (and @12xx12 also had a note there before editing it :upside_down_face: ) the problem was/is that cuberite does not really have a notion of releases as it's mainly tracking features/versions of vanilla, not making new features on it's own.
> 
> Unless you want to spam every new commit in master to twitter then one idea would be to have everything "hot" go into some kind of `devel` branch and only do devel -> master once in a while (after ""testing"" it) and push that as a tweet.

so... so releases?

**Comment by mathiascode** [comment]
@bearbin Are you the only person with access to the Twitter account nowadays? Perhaps access could be granted to more people interested in tweeting about new features, if candidates exist.

**Comment by 12xx12** [comment]
Maybe we make the website post a tweet if there is a new news "article" availeable

**Comment by bearbin** [comment]
@12xx12 's suggestion of posting whenever there's a new article on the website is a good idea, I'll try and keep that going if I remember.

@mathiascode I do have access to the account, not sure who else also has access if anyone, I would be happy to share the details with anyone on the team who wants to keep it up to date, but I don't want a load of junk going on it - every commit would be a partoicularly useless thing to post there.

**Comment by 12xx12** [comment]
Maybe there is a way to include a automatic tweet when generating the HTML code from the markdown code in the news article. So we don't have to manually do that.

Because everyone can open a pr we can have different people doing this and we only have to review the PR's and that's it

**Comment by koutsie** [comment]
Sounds good?
